hasjob/templates/detail.html

```diff
@@ -24,6 +24,9 @@
   {%- if post.is_old() %}
     <meta name="robots" content="nofollow" />
   {%- endif %}
+  {%- if is_post_admin %}
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/c3/0.4.10/c3.css">
+  {%- endif %}
```

Comment:
Doesn't look like it. Just removed the JS loading from bottom and the chart broke.